x11-libs/xbae/xbae-4.60.4-r2.ebuild

```diff
@@ -29,8 +33,18 @@ PATCHES=(
 	"${FILESDIR}"/${P}-tmpl.patch
 	"${FILESDIR}"/${P}-lxmp.patch
 	"${FILESDIR}"/${P}-Makefile.in.patch
+	# https://sourceforge.net/p/xbae/patches/3/
+	"${FILESDIR}"/${P}-configure-c99.patch
+	"${FILESDIR}"/${P}-modern-autotools.patch
+	"${FILESDIR}"/${P}-c99.patch
+	"${FILESDIR}"/${P}-c23.patch
```

Comment:
The PR adds one patch, but four are listed here, where do the other three come from?

Reply:
To answer both this, and previous question: From -r1, the stable ebuild.

Reply:
Right, sorry, I was clearly too sleepy when looking at this before.